dev-docs/adding-data/adding_test_data.md

```diff
@@ -82,13 +88,14 @@ from hab studio:
 
 ensure you have `jq` installed
 
-option one: `infra_service_load_sample_data`
-By default, 50 records of the infra servers and orgs will be added with the prefix `chef-server` and `chef-org` respectively.
-
-option two: `infra_service_load_sample_data -N 100 -S infra-server -O infra-org`
-100 records of the infra servers and orgs will be added, modify the numbers on each of those options to meet your needs.
-
-option three: look through the pinned items in slack channel #automate-infra-views and follow the instructions there (works against running chef server)
+there are three options...
+1. `infra_service_load_sample_data`
+    
+    By default, 50 records of the infra servers and orgs will be added with the prefix `chef-server` and `chef-org` respectively.
+1. `infra_service_load_sample_data -N 100 -S infra-server -O infra-org`
+    
+    100 records of the infra servers and orgs will be added, modify the numbers on each of those options to meet your needs.
+1. [internal steps](https://chefio.slack.com/archives/CPYNX33LJ/p1583485764104300) this is the only method that works against a running chef server and will allow you to get data that does not result in the cookbook error
```

Comment:
Is "the cookbook error" what I referred to? If so, it is actually an org error; it will happen with any of cookbook's siblings, too (environment, data bags, etc.)--all because they are children of the org. Might mention the error phrase, too: "invalid org ID: private key block size invalid"

Reply:
@msorens `invalid org ID: private key block size invalid` would be the better user-readable message in future As I am correctly working on.

will commit changes to mentioned `(environment, data bags, etc.)` as well

thanks